Python/Product/Cookiecutter/View/CookiecutterSearchPage.xaml

```diff
@@ -146,7 +145,8 @@
                 <HierarchicalDataTemplate DataType="{x:Type vm:CategorizedViewModel}" ItemsSource="{Binding Path=Templates}">
                     <TextBlock Margin="0 4 0 4"
                                FontWeight="Bold"
-                               Text="{Binding Path=DisplayName}"/>
+                               AutomationProperties.ItemType="Category"
```

Comment:
Looks like "Category" should be localized?  It gives examples of "Document File" and "Folder".
https://msdn.microsoft.com/en-us/library/system.windows.automation.automationproperties.itemtype(v=vs.110).aspx